cicd/pipelines/understanding-openshift-pipelines.adoc

```diff
@@ -19,6 +19,9 @@ toc::[]
 * You can use {pipelines-title} to build images with Kubernetes tools such as Source-to-Image (S2I), Buildah, Buildpacks, and Kaniko that are portable across any Kubernetes platform.
 * You can use the {product-title} Developer console to create Tekton resources, view logs of pipeline runs, and manage pipelines in your {product-title} namespaces.
 
+//Brief overview
+include::modules/op-overview.adoc[leveloffset=+1]
```

Comment:
Per the OCP doc guidelines, "overview" shouldn't be used in headings: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings

Maybe you could combine this with the content above the "Key features" heading rather than needing it to be a separate module?

Reply:
"Key features" is a bullet list format (in the assembly, before modules) common to several assemblies the existing doc structure, and I don't want to touch it before i can restructure. I was not aware of the "overview" ban (which makes no sense to me) and I'll try to conjure a replacement.